src/material-experimental/mdc-paginator/paginator.html

```diff
@@ -27,7 +27,7 @@
     </div>
 
     <div class="mat-mdc-paginator-range-actions">
-      <div class="mat-mdc-paginator-range-label">
+      <div class="mat-mdc-paginator-range-label" aria-live="polite">
```

Comment:
One question here: what would happen if the user landed on a page with a bunch of paginators? Would their range labels all be queued up and announced on init or is this only going to fire when the text changes?

Reply:
With the native `aria-live` they should be queued up.